static/builder.js

```diff
@@ -1451,8 +1495,16 @@ function displaySearchResultsAsync(items, start, div) {
             if (item.enhancements || itemInventory && itemInventory.enchantments && itemInventory.enchantments[item.id]) {
                 html += " enhanced";
             }
-            html += '" onclick="fixItem(\'' + item.id + '\', ' + currentItemSlot + ', ' + enhancementString + ')">';
-            html += displayItemLine(item);
+            
+            if(searchClickBehavior == ClickBehaviors.EQUIP) {
+                html += '" onclick="fixItem(\'' + item.id + '\', ' + currentItemSlot + ', ' + enhancementString + ')">';
+            } else {
+                html += '" >';
```

Comment:
Why not take the opportunity to allow to click on the whole line to exclude/include back on item when in the ClickBehaviors.EXCLUDE mode ?

Reply:
Great suggestion and easy to add, will fix.